pkg/cri/sbserver/images/image_pull.go

```diff
@@ -212,9 +212,10 @@ func (c *CRIImageService) PullImage(ctx context.Context, r *runtime.PullImageReq
 		}
 	}
 
+	const mbToByte = 1024 * 1024
 	size, _ := image.Size(ctx)
 	imagePullingSpeed := float64(size) / time.Since(startTime).Seconds()
-	imagePullThroughput.Observe(imagePullingSpeed)
+	imagePullThroughput.Observe(imagePullingSpeed / mbToByte)
```

Comment:
```
	imagePullingSpeed := float64(size) / mbToByte / time.Since(startTime).Seconds() 
	imagePullThroughput.Observe(imagePullingSpeed)
```

To make the formula more readable, I prefer to alter to something like above.